crates/cxx-qt-lib-headers/include/core/qcoreapplication.h

```diff
@@ -127,5 +130,14 @@ qapplicationRemoveLibraryPath(T& app, const QString& path)
   app.removeLibraryPath(path);
 }
 
+#ifdef CXX_QT_GUI_FEATURE
+template<typename T>
+QFont
+qapplicationFont(const T& app)
```

Comment:
why is this still a template and not in qguiapplication like setFont (or the opposite).